start.go

```diff
@@ -202,8 +202,7 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 				"--state-dir", root,
 				"--listen", filepath.Join(namespace, "namespaced.sock"),
 			)
-			cmd = exec.Command("runv", args...)
-			cmd.Path = path
+			cmd = exec.Command(path, args...)
```

Comment:
@laijs yeah, we've already gurranteed the path is correct before, I'm OK with both, prefer this one.
```
cmd := &exec.Cmd{
            Path: path,
            Args: append([]string{"runv"}, args...),
        }
```
@YaoZengzeng 
> `cmd, err := exec.LookPath(QEMU_SYSTEM_EXE)` 

Also need $PATH to find the abs path if `QEMU_SYSTEM_EXE` is not. 
FYI: https://golang.org/src/os/exec/lp_unix.go